src/main/java/org/jenkinsci/plugins/JiraTestResultReporter/JiraUtils.java

```diff
@@ -122,18 +126,59 @@ public static String createIssueInput(AbstractProject project, TestResult test,
      * @param project the project
      * @param test the test
      * @param envVars the environment variables
+     *  @fields limited set of fields to be returned to optimize the performance issues
      * @return a SearchResult. Empty SearchResult means nothing was found.
      */
     public static SearchResult findIssues(AbstractProject project, TestResult test, EnvVars envVars)
     {
         String projectKey = JobConfigMapping.getInstance().getProjectKey(project);
         FieldInput fi = JiraTestDataPublisher.JiraTestDataPublisherDescriptor.templates.get(0).getFieldInput(test, envVars);
         String jql = String.format("status != \"closed\" and project = \"%s\" and text ~ \"%s\"", projectKey, escapeJQL(fi.getValue().toString()));
```

Comment:
I think we last suggested that we should also add username to this query. What do you think?